src/view/placeholder.js

```diff
@@ -50,11 +57,16 @@ export function detachPlaceholder( view, element ) {
 	const document = element.document;
 
 	if ( documentPlaceholders.has( document ) ) {
+		const info = documentPlaceholders.get( document ).get( element );
+		if ( info.removeCkClass ) {
+			view.change( writer => writer.removeClass( 'ck', element ) );
+		}
+
 		documentPlaceholders.get( document ).delete( element );
 	}
 
 	view.change( writer => {
```

Comment:
We have two `view.change()` calls which mean rendering the thing twice. Can't we have one big `view.change()`? It doesn't actually matter, but I'm asking out of curiosity.